src/components/v5/common/EmptyContentBox/EmptyContentBox.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
```

Comment:
we have prepared empty state component in stage 1, please reuse it and just change styles